<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: rgb(115, 248, 255);
color: rgb(137, 218, 255);
background-image: url(:/ressources/drew coffman-unsplash.jpg);
font: 75 10pt &quot;Trebuchet MS&quot;;

border-bottom-color: rgb(132, 235, 235);
background-color: rgb(255, 180, 185);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Personnels</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>931</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 160, 165);
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Consulter la liste des personnels</string>
      </attribute>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>220</y>
         <width>201</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/ressources/empl.jpg);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifierperso">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>120</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
       </property>
       <property name="text">
        <string>Modifier </string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimerperso">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>70</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
       </property>
       <property name="text">
        <string>Supprimer </string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="graphicsView_5">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>30</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/ressources/business_management_discovery_search_focus_research-512.png);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_suppcin">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>170</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color : rgb(255, 84, 130);
font: 9pt &quot;Snap ITC&quot;;
</string>
       </property>
       <property name="text">
        <string>CIN :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_supp">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>170</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(74, 74, 74);
font: 9pt &quot;Snap ITC&quot;;</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
      <widget class="QTableView" name="tablePerso">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>651</width>
         <height>291</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="rechercher">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>30</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Ajouter un personnel </string>
      </attribute>
      <widget class="QLabel" name="label_Cin">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;
color: rgb(68, 68, 68);

</string>
       </property>
       <property name="text">
        <string>CIN :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nom">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>55</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;
color: rgb(68, 68, 68);
</string>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_prenomp">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;
color: rgb(68, 68, 68);
</string>
       </property>
       <property name="text">
        <string>Prenom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_datenp">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>180</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;
color: rgb(68, 68, 68);
</string>
       </property>
       <property name="text">
        <string>Date_de_naissance :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_tel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>230</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;
color: rgb(68, 68, 68);
</string>
       </property>
       <property name="text">
        <string>Tel :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_mail">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>270</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;
color: rgb(68, 68, 68);
</string>
       </property>
       <property name="text">
        <string>Mail :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nbmar">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>310</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;
color: rgb(68, 68, 68);
</string>
       </property>
       <property name="text">
        <string>Nb_Mar_Org :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prenomp">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;Snap ITC&quot;;

color: rgb(56, 181, 200);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_datenp">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>180</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;Snap ITC&quot;;

color: rgb(56, 181, 200);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nomp">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;Snap ITC&quot;;

color: rgb(56, 181, 200);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Cin">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;Snap ITC&quot;;

color: rgb(56, 181, 200);

</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_tel">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>230</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;Snap ITC&quot;;

color: rgb(56, 181, 200);</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_mail">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>270</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;Snap ITC&quot;;

color: rgb(56, 181, 200);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nbmar">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>310</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;Snap ITC&quot;;

color: rgb(56, 181, 200);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouterperso">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>170</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
       </property>
       <property name="text">
        <string>Confirmer</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tâches</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>931</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 160, 165);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Consulter la liste des tâches </string>
      </attribute>
      <widget class="QPushButton" name="modifierTache">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>180</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>130</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
       </property>
       <property name="text">
        <string> E-mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_suppnomT">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>280</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color : rgb(255, 84, 130);
font: 9pt &quot;Snap ITC&quot;;</string>
       </property>
       <property name="text">
        <string>NomT :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_suppmodT">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>280</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimertache">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>240</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableTache">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>581</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="export_excel">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>80</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
       </property>
       <property name="text">
        <string>export excel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Ajouter une tâche</string>
      </attribute>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>55</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;
color: rgb(68, 68, 68);
</string>
       </property>
       <property name="text">
        <string>NomT:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;
color: rgb(68, 68, 68);
</string>
       </property>
       <property name="text">
        <string>EtatT :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>210</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;
color: rgb(68, 68, 68);
</string>
       </property>
       <property name="text">
        <string>DateLimiteT :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_NomTache">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">

font: 8pt &quot;Snap ITC&quot;;

color: rgb(56, 181, 200);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_EtatT">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;Snap ITC&quot;;

color: rgb(56, 181, 200);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_DateLimite">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>210</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;Snap ITC&quot;;

color: rgb(56, 181, 200);
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter_tache">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>160</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
       </property>
       <property name="text">
        <string>Confirmer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>260</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Snap ITC&quot;;
color: rgb(68, 68, 68);</string>
       </property>
       <property name="text">
        <string>IDP :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_CinPTache">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>260</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;Snap ITC&quot;;

color: rgb(56, 181, 200);</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Capteur de gaz</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Snap ITC&quot;;

color: rgb(68, 68, 68);</string>
     </property>
     <property name="text">
      <string>Etat de buzzer et led rouge :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ON">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>150</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 160, 165);
font: 9pt &quot;Snap ITC&quot;;
</string>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OFF">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>150</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 160, 165);
font: 9pt &quot;Snap ITC&quot;;
</string>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
